src/OpenRiaServices.Server/Test/OpenRiaServices.Server.Test.csproj

```diff
@@ -4,8 +4,12 @@
     <DefineConstants>$(DefineConstants);SERVERFX</DefineConstants>
     <TargetFramework>net472</TargetFramework>
   </PropertyGroup>
+    <PropertyGroup>
+        <GenerateRuntimeConfigurationFiles>True</GenerateRuntimeConfigurationFiles>
```

Comment:
what does this ssetting do ?

Reply:
It generates files that specific .net Core runtime settings and version. Removed since we do not need it for net472.
https://stackoverflow.com/questions/44174407/what-is-the-purpose-of-msbuilds-generateruntimeconfigurationfiles